tests/fe/bdm_16.cc

```diff
@@ -0,0 +1,63 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2003 - 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// the test is used to check the restriction_is_additive flags
+
+#include "../tests.h"
+#include <deal.II/base/logstream.h>
+#include <deal.II/fe/fe_bdm.h>
+
+#include <fstream>
+#include <string>
+
+
+std::ofstream logfile ("output");
+
+template<int dim>
+void
+test (const unsigned int degree)
+{
+  FE_BDM<dim> fe_bdm(degree);
+
+  deallog << "Degree=" << degree
```

Comment:
oh, i see. Then please elaborate it in the in-code comment at the beginning of the test file so that if someone looks at this file in a few years, it would be easier to understand why the test is here.